client/src/main/java/io/micrometer/prometheus/rsocket/PrometheusRSocketClient.java

```diff
@@ -29,6 +29,8 @@
 import org.xerial.snappy.Snappy;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
+import reactor.util.Logger;
+import reactor.util.Loggers;
```

Comment:
I'm not sure if you intended to use this Logger. We should probably use the Micrometer one.